modules/sno-du-configuring-sriov.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-sriov_{context}"]
+= Configuring single root I/O virtualization (SR-IOV)
+
+SR-IOV is commonly used to enable the fronthaul and the midhaul networks.
+
+Procedure
+
+* Use the following configuration to configure SRIOV on a single node distributed unit (DU) using an
+Intel NIC. Note that the first custom resource (CR) is required. The following CRs are examples.
```

Comment:
Leave out the reference to the Intel NIC and use a callout on the deviceType (lines 46 and 73) that indicates the user can select between vfio-pci and netdevice as needed.

Reply:
Deleted " using an Intel NIC". Added callouts.